catalog/users/hitachi.json

```diff
@@ -0,0 +1,8 @@
+{
+    "id": "hitachi",
+    "kind": "solutionprovider",
+    "name": "Hitachi, Ltd.",
+    "email": "",
+    "website": "https://www.hitachi.com/",
+    "logo": "Logo data is uploaded to Github. (pact-catalog/users/hitachi-cslogo.zip)"
```

Comment:
The `"logo"` field only accepts a URL as its value. Can you please replace the current string by one containing a URL to your logo (hosted somewhere else)? 
Thank you!

Reply:
Dear @raimundo-henriques and @prasadt1,
As a result of our internal review, we will temporarily register without the logo. As soon as the internal procedures are completed so that the URL can be obtained, we will update solution.json that includes the URL of the logo.
Also, sorry, we can not prepare the official email for EcoAssist-Pro/LCA. However an user considering to use EcoAssist can access through a form on our official web site.
Thank you.